.nvmrc

```diff
@@ -1 +1 @@
-14.17.6
```

Comment:
This version was not compatible with `package-lock.json` file, I tried upgrading to Node 18 but the app won't run on it.

Reply:
I created this issue (https://github.com/DjedAlliance/Djed-Solidity-WebDashboard/issues/75) to address this.